library.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "compact25519",
+    "version": "1.0.0",
+    "repository": { "type": "git", "url": "https://github.com/DavyLandman/compact25519"},
```

Comment:
can we at least have this point towards a tag? or does platformio take care of this themselves?

Reply:
Okay, yes, I only wanted to make sure nobody would get an unreleased version. (like a user getting `main` instead of a released version)